packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1261,6 +1261,9 @@ class SelectionRect {
 
 /// An interface to receive granular information from [TextInput].
 ///
+/// Mixin with [TextInputClient] to receive any default implementations from
```

Comment:
That's only true if the class is not already extending `TextInputClient` right? Then it might be clearer to state the limitation and explain what to do if you see the error message.